back-office/multistoregeneralspecs.md

```diff
@@ -0,0 +1,201 @@
+
+The name of the selected context is displayed, it can be:
+- All shops
+- A shop group name
+- A shop name
+
+**View my shop link**
+
+When a shop is selected, a link "view my shop" is displayed and opens the front-office of the shop.
+
+**Context drop-down**
+
+In the multistore header, there is a context drop-down allowing to select the context.
+
+In the drop-down there is a search bar with auto-completion allowing to search and select a shop or a group quickly.<br/>
+There is a placeholder: "Search shop name"<br/>
+As soon as a user types a character in the field, a search is immediately performed and returned.<br/>
+The results are displayed in a frame below the search field. <br/>
+The displayed results can then be browsed, either by the arrow keys on the keyboard, or by the mouse cursor.<br/>
+If the user chooses one of the listed results, it is automatically written in the search field and selected as context.
+
+The options listed in this drop-down are:
+- All shops with a link to select it as context
+- The shop groups names with a link to select the group as context
+- The shop names with a link to select the shop as context
+
+The colors of each shop and group are displayed before each option.<br/>
+When the user hovers the color, a tooltip is displayed "edit color"<br/>
+The user can click on the color to change it<br/>
+It opens the store's or group's page edition.
+
+If the shop URL is configured, a "view my shop" link is displayed after each shop name.<br/>
+It opens the front-office of the corresponding shop.
+
+If the shop URL isn't configured, a "configure shop URL" link is displayed after each shop name.<br/>
+It opens the store's page edition.
+
+
+### Shop association block
+
+The shop association block is displayed when you add or edit something.<br/>
+The shop association block displays all groups and all shops.<br/>
+Checkboxes allow to select / unselect groups and shops.<br/>
+It's possible to expand / collapse the wall tree and expand / collapse each group<br/>
+If you select / unselect a group, all shops of this group are selected / unselected
+
+The shop association block allows to choose for which store(s) you want to associate or disassocate the current element.
+
+When you add something, the shop(s) selected in the context are automatically checked in the shop association block.<br/>
+But it's possible to select / uncelect shops, no matter the context selected.
+
+When you edit something, you can unselect shop(s) previously selected and therefore disassocate it from these shops.<br/>
+And you can also, select shop(s) not previously selected and therefore associate it with these shops.<br/>
+If the merchant unselect all the checkboxes (all the shops), then an alert message in red is displayed above or below the shop association block to inform of the obligation to have at least one shop checked: "You should select at least one shop to be able to save" + the save button is disabled. **only from 1.7.8**
```

Comment:
To be confimed by @TristanLDD if it's above or below
@LouiseBonnard let me know if the message is clear enough ?

Reply:
I would write _You have to select at least one shop to associate this item with._ based on the wording [here](https://docs.google.com/document/d/1Ag-KUkBpYgnEgETdZ4Cvb-zbcbgIBcAU44Na8b8-83Q/edit) (search by 'You have to').

Also, I would recommend this template for all alert messages throughout the feature: _You have to select [a shop] to [do something]._ in _Admin.Notifications.Error_ - which brings me to wonder: is it an error (after a wrong action) or a warning (before an action that might be wrong)?

Reply:
@LouiseBonnard ok for the wording
it's an error displayed when the user unselect all the checkboxes and before the save

Reply:
So it is more like a warning, right?